spec/abstractops.html

```diff
@@ -1421,31 +1421,31 @@ <h1>
       1. Let _duration_ be ParseText(StringToCodePoints(_isoString_), |TemporalDurationString|).
       1. If _duration_ is a List of errors, throw a *RangeError* exception.
       1. Let each of _sign_, _years_, _months_, _weeks_, _days_, _hours_, _fHours_, _minutes_, _fMinutes_, _seconds_, and _fSeconds_ be the source text matched by the respective |Sign|, |DurationYears|, |DurationMonths|, |DurationWeeks|, |DurationDays|, |DurationWholeHours|, |DurationHoursFraction|, |DurationWholeMinutes|, |DurationMinutesFraction|, |DurationWholeSeconds|, and |DurationSecondsFraction| Parse Node contained within _duration_, or an empty sequence of code points if not present.
-      1. Let _yearsMV_ be ? ToIntegerWithTruncation(CodePointsToString(_years_)).
-      1. Let _monthsMV_ be ? ToIntegerWithTruncation(CodePointsToString(_months_)).
-      1. Let _weeksMV_ be ? ToIntegerWithTruncation(CodePointsToString(_weeks_)).
-      1. Let _daysMV_ be ? ToIntegerWithTruncation(CodePointsToString(_days_)).
-      1. Let _hoursMV_ be ? ToIntegerWithTruncation(CodePointsToString(_hours_)).
+      1. If _yearsMV_ is *undefined*, let _yearsMV_ be *+0*<sub>𝔽</sub>; else let _yearsMV_ be ? ToIntegerWithTruncation(CodePointsToString(_years_)).
+      1. If _monthsMV_ is *undefined*, let _monthsMV_ be *+0*<sub>𝔽</sub>; else let _monthsMV_ be ? ToIntegerWithTruncation(CodePointsToString(_months_)).
+      1. If _weeksMV_ is *undefined*, let _weeksMV_ be *+0*<sub>𝔽</sub>; else let _weeksMV_ be ? ToIntegerWithTruncation(CodePointsToString(_weeks_)).
+      1. If _daysMV_ is *undefined*, let _daysMV_ be *+0*<sub>𝔽</sub>; else let _daysMV_ be ? ToIntegerWithTruncation(CodePointsToString(_days_)).
+      1. If _hoursMV_ is *undefined*, let _hoursMV_ be *+0*<sub>𝔽</sub>; else let _hoursMV_ be ? ToIntegerWithTruncation(CodePointsToString(_hours_)).
```

Comment:
I think these lines can now be left as they were. If the Parse Nodes are not present, then the variables e.g. _years_ are an empty sequence of code points (step 3). CodePointsToString of an empty sequence of code points returns an empty string, and ToIntegerWithTruncation of an empty string returns 0.

(same for the minutes, seconds, etc. lines below)